pyuvdata/uvdata/uvdata.py

```diff
@@ -300,13 +300,14 @@ def __init__(self):
             '"Stokes", but this is inaccurate as visibilities cannot be in '
             "true Stokes polarizations for physical antennas. We adopt the "
             "term pseudo-Stokes to refer to linear combinations of instrumental "
-            "visibility polarizations (e.g. pI = xx + yy)."
+            "visibility polarizations (e.g. pI = xx + yy). A value of 0 is only "
```

Comment:
What does the value "0" represent?

Reply:
It indicates that this is a flex-pol object, which means that the spectral windows have different polarizations (which is not normally allowed) but each spectral window only has one polarization. This mode was developed to support the SubMillimeter array, which can operate in such a mode, but it turns out that we're going to use it in HERA as well as it can significantly speed up single polarization reads. There are methods for converting HERA-type data between the regular layout (single SPW with all pols) and the flex-pol layout (multiple SPWs each with one pol).

Can you suggest some language that would make this more clear to you and other users?